### PR TITLE
Add ships.json sample file

### PR DESCRIPTION
**Pull request by royfalk**
This file to eventually contain all ships from units.json
Related to [#739](https://github.com/vegastrike/Vega-Strike-Engine-Source/pull/739)

Please answer the following:

Code Changes:
- [x] Have the PR Validation Tests been run? See https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Pull-Request-Validation

Issues:
- See other MR

### CODE REVIEW

**Review by stephengtuggy — CHANGES_REQUESTED** [review]
Need to add the new ships.json file to CMakeLists.txt. This will need to be done every time we add asset files.

**Comment by royfalk** [comment]
> 



> 

We've not done so for units.json or units.csv and really all other assets.
We've only done so for unit.json for testing.

Did you mean 'every time we add asset files to testing'?

File: units/ships.json [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+                "Forward_Accel": "15000",
+                "Retro_Accel": "10000",
+                "Left_Accel": "4000",
+                "Right_Accel": "4000",
+                "Top_Accel": "4000",
+                "Bottom_Accel": "4000",
+                "Afterburner_Speed_Governor": "220",
+                "Default_Speed_Governor": "220",
+                "Hold_Volume": "50",
+                "Prohibited_Upgrades": "upgrades/Overdrive:1;upgrades/Capacitors:1;upgrades/Reactors:1;upgrades/Armor:1;upgrades/Sensors:1;upgrades/Shield_Systems:1;upgrades/Passenger_Quarters:1;upgrades/Repair_Systems:1;upgrades/ECM_Systems:1",
```
Comment (BenjamenMeyer) [inline units/ships.json:56]: It would be good to break these down as well. guessing that needs another round of refactor so not blocking, just noting.

**Comment by BenjamenMeyer** [comment]
@royfalk we need to add the JSON files - and presumably should have added the CSV files - into the CMake data so it gets installed properly; similar to how the documentation/manpages are added.

**Comment by stephengtuggy** [comment]
@royfalk The last couple of files we added to the assets, I added to CMake for you, in followup PR's. :-) It definitely does need to be done.

**Review by stephengtuggy — APPROVED** [review]
Looks good to me

**Pull request merged**